Samples/OpenQasmReader.Tests/Validate/CNot.qs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.Quantum.Samples.OpenQasmReader.Tests.Validate
```

Comment:
these are all missing copyright headers... I understand why, but not sure if that is problematic. let me check.

Reply:
Legal will probably take more effort than adding a bit of ignore code in there. Added workaround.